src/parser/jobs/mnk/modules/MissedBrotherhood.tsx

```diff
@@ -0,0 +1,85 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {dependency} from 'parser/core/Injectable'
+import {EvaluatedAction, RaidBuffWindow, WindowEvaluator} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {HistoryEntryPredicate} from 'parser/core/modules/ActionWindow/windows/ActionWindow'
+import {SEVERITY, SeverityTiers, Suggestion, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import {Message} from 'semantic-ui-react'
+import {Brotherhood} from './Brotherhood'
+import {RiddleOfFire} from './RiddleOfFire'
+
+const MISSED_BROTHERHOOD_TIERS: SeverityTiers = {
+	1: SEVERITY.MEDIUM,
+	2: SEVERITY.MAJOR,
+}
+
+class MissedBrotherhoodEvaluator implements WindowEvaluator {
+	private icon: string
+	constructor(icon: string) {
+		this.icon = icon
+	}
+
+	public suggest(windows: Array<HistoryEntry<EvaluatedAction[]>>): Suggestion | undefined {
+		if (windows.length === 0) {
+			return undefined
+		}
+
+		return new TieredSuggestion({
+			icon: this.icon,
+			content: <Trans id="mnk.brotherhood.missed-window.content">
+				Try and make sure your <DataLink action="BROTHERHOOD"/> casts line up with your <DataLink action="RIDDLE_OF_FIRE"/> windows to maximize buff stacking.
+			</Trans>,
+			why: <Trans id="mnk.brotherhood.missed-window.why">
+				<Plural value={windows.length} one="# cast" other="# casts"/> of <DataLink action="BROTHERHOOD"/> did not occur during a <DataLink status="RIDDLE_OF_FIRE"/> window.
+			</Trans>,
+			tiers: MISSED_BROTHERHOOD_TIERS,
+			value: windows.length,
```

Comment:
Same nit as the brotherhood.missed-players suggestion.
```suggestion
			icon: this.icon,
			content: <Trans id="mnk.brotherhood.missed-window.content">
				Try and make sure your <DataLink action="BROTHERHOOD"/> casts line up with your <DataLink action="RIDDLE_OF_FIRE"/> windows to maximize buff stacking.
			</Trans>,

			</Trans>,
			tiers: MISSED_BROTHERHOOD_TIERS,
			value: windows.length,
                        why: <Trans id="mnk.brotherhood.missed-window.why">
				<Plural value={windows.length} one="# cast" other="# casts"/> of <DataLink action="BROTHERHOOD"/> did not occur during a <DataLink status="RIDDLE_OF_FIRE"/> window.			
```

Reply:
Fixed.